config/redis.codeship.example

```diff
@@ -8,3 +8,5 @@ redis_autocomplete:
   test: localhost:6381
 redis_rollout:
   test: localhost:6381
+redis_autocomplete:
```

Comment:
This file already has redis_autocomplete.

Reply:
Okay, that's... odd. It's not showing up on my local copy, but I see it here. o_0